CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### App Center
 
 * **[Improvement]** Use namespaced `NSUserDefaults` keys with the **MSAppCenter** prefix for all the keys set by the SDK. Fixed a few keys missing namespace.
+* **[Feature]** Partial Swift Package Manager distribution support added (for Core and Analytics modules).
```

Comment:
To me, it sounds better 
```suggestion
* **[Feature]** Swift Package Manager support for Core and Analytics modules.
```

Reply:
I'd move it outside of this subsection

Reply:
I don't think it should be somewhere else but this subsection - It doesn't fit any module's section as well as it fits general.
Updated phrase.